alvr/session/src/settings.rs

```diff
@@ -1773,6 +1775,8 @@ pub fn session_settings_default() -> SettingsDefault {
                         body_tracking_bd: SwitchDefault {
                             enabled: true,
                             content: BodyTrackingBDConfigDefault {
+                                body_tracking: true,
+                                object_tracking: false,
```

Comment:
Object tracking should default to true, like body_tracking. You said that you select which to enable in the system settings, and they are mutually exclusive. I might be still misunderstanding things, but doesn't either body tracking or object tracking fail to initialize or to poll tracking data because it is disabled in the system settings? so if we attempt to use both we have the certainty that at least one works

Reply:
`get_motion_tracker_locations` works anyway ignoring current mode setting.

Reply:
Oh hm. But what do you mean with "works"? Can you poll the data but it would be null if the user sets body tracking mode, or polling always returns valid data?

Reply:
It always returns valid data regardless of the user setting, yeah.

Reply:
In that case, let's go back to your idea of having mutually exclusive body and object tracking, using an enum. Sorry for the confusion.